Drivers/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required(VERSION 3.10)
+enable_language(ASM)
+set(NCT_PROJECTS $ENV{NCT_PROJECTS})
+
+set(COMPILE_OPTIONS
+    -march=armv4 -mbig-endian
+    $<$<COMPILE_LANGUAGE:ASM>:
+        -I ${CMAKE_SOURCE_DIR}>
+    $<$<COMPILE_LANGUAGE:C,CXX>:
+        -O2
```

Comment:
Are you replacing `-O3 -Os` with `-O2` here?

Reply:
Good catch, I changed it back to what the Makefile uses.